index.js

```diff
@@ -120,7 +120,11 @@ const pollWorkflow = () => {
       headers: headers,
     })
     .then((response) => {
-      if (response.data.items[0].status != "running") {
+      if (
+        response.data.items[0].status != "running" ||
```

Comment:
I think you wanted boolean &&

"If the status isn't running, and isn't not_run, and isn't on_hold..."

I think you could also one-line this by using the `includes` method:
```typescript
if( ! ["not_run", "on_hold", "running"].includes(response.data.items[0].status) ) {
```